src/templates/_macros/app_tile.html

```diff
@@ -12,6 +12,7 @@
       src -- if tile is a link, attach a src param for analytics purposes
   #}
   {% set app = apps.transform(app) %}
+  {% set name = app.name if is_detail else app.short_name %}
```

Comment:
less performant, less out-of-the-box/clean though. nunjucks outputs more code than pure JS would